openedx/core/djangoapps/ace_common/utils.py

```diff
@@ -0,0 +1,25 @@
+"""
+Utility functions for edx-ace.
+"""
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def setup_firebase_app(firebase_credentials, app_name='fcm-app'):
+    """
+    Returns a Firebase app instance if the Firebase credentials are provided.
+    """
+    try:
+        import firebase_admin  # pylint: disable=import-outside-toplevel
+    except ImportError:
+        log.error('Could not import firebase_admin package.')
+        return
+
```

Comment:
I think this is a core import that should be kept is a real import since the package is installed always in the requirements.

```suggestion
import firebase_admin  # pylint: disable=import-outside-toplevel
```

Reply:
Fixed.